src/services/dataService/__tests__/user.test.js

```diff
@@ -1,14 +1,45 @@
 const test = require('../../../test')
 const seedData = require('../../../test/seedData')
-const {getUser, getList, findLists} = require('../../dataService')
+const {getUser, findUsers, createUser, deleteUser, updateUser} = require('../../dataService/user')
 
 test('dataService', function(t) {
   return seedData().then(() => {
     t.test('Get user test', function (st) {
       getUser(1).then(function (user) {
-        st.equal(user.id, 1, 'Fuck yeah!' )
+        st.equal(user.id, 1, 'Fuck yeah!')
         st.end()
       })
     })
+
+    t.test('Find user test', function (st) {
+    	findUsers().then(function (user) {
+    		st.ok(user.length > 0, 'Find a users name, email and password')
+    		st.end()
+    	})
+    })
+
+    t.test('Create user test', function (st) {
+    	createUser('michael', 'example@.com', '123').then(function (user) {
```

Comment:
Better alternative for createUser() signature: accept one argument for all new item properties.

Example:
```js
createUser({name: 'michael', email: 'example@.com', password: '123'})
```

This is much more flexible as your model changes/grows over time.